.github/workflows/release.yml

```
@@ -0,0 +1,144 @@
+      - name: Install Dependencies
+        run: |
+          cd web
+          npm ci
+
+      - name: Build Frontend
+        run: |
+          cd web
+          npm run build
+
+      - name: Upload Artifact
+        uses: actions/upload-artifact@v3
```

actionlint (1.7.4)

the runner of "actions/upload-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)